etc/library.sh

```diff
@@ -101,6 +106,21 @@ function configure_app()
   return $ret
 }
 
+function set-nc-domain() {
+  local DOMAIN="${1?}"
+  URL="https://${DOMAIN%*/}"
+  # trusted_domain no 3 will always contain the overwrite domain
+  [[ "$2" == "--no-trusted-domain" ]] || ncc config:system:set trusted_domains 3 --value="${DOMAIN%*/}"
+  ncc config:system:set overwrite.cli.url --value="${URL}/"
+  if ncc app:enable -q notify_push
```

Comment:
If we're using the wrong nc version, this if condition evaluates to false

Reply:
Btw, I tested that it only fails if the app *can't* be enabled (it succeeds even if it was already enabled)

Reply:
I've used something like `ncc | grep -q notify_push` in the past, but whatever works is fine